utils/segprint.py

```diff
@@ -222,18 +398,25 @@ def tile_segnames(tiles):
     return ret
 
 
+def load_tiles():
+    # TODO: Migrate to new tilegrid format via library.
+    with open("%s/%s/tilegrid.json" % (os.getenv("XRAY_DATABASE_DIR"),
+                                       os.getenv("XRAY_DATABASE")), "r") as f:
+        tiles = json.load(f)
+    return tiles
+
+
 def run(
-        db_root,
         bits_file,
         segnames,
         omit_empty_segs=False,
         flag_unknown_bits=False,
         flag_decode_emit=False,
         flag_decode_omit=False,
         verbose=False):
-    tiles, segments = mk_grid(db_root)
-    db = prjxraydb.Database(db_root)
-
+    db = prjxraydb.Database()
```

Comment:
Both mk_grid and Database rely on a common db_root, this should be made explicit.